src/main/java/com/google/maps/internal/OkHttpPendingResult.java

```diff
@@ -216,9 +205,9 @@ private T parseResponse(OkHttpPendingResult<T, R> request, Response response) th
 
     // Places Photo API special case
     if (contentType != null &&
-        contentType.startsWith("image") &&
-        responseClass == PhotoRequest.Response.class &&
-        response.code() == 200) {
+            contentType.startsWith("image") &&
```

Comment:
Please put back to 4 space indent: https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation